cluster/run.sh

```diff
@@ -32,7 +32,7 @@ pip install numpy --upgrade --user
 popd
 
 # Run job
```

Comment:
Rather than hardcode this in the run script, I'd move to an argument that gets passed in to the qsub subcommand.